src/pages/CustomerInvoicePage.js

```diff
@@ -251,6 +250,19 @@ const reducer = (state, action) => {
       }
       return { ...state, dataState: newDataState };
     }
+    case 'sortBy': {
+      const { data, isAscending } = state;
+      const { sortBy } = action;
+      const columnKeyToDataType = {
```

Comment:
Just an observation: Probably slightly more optimal to declare this outside the reducer, but I approve the convenience of having it right where it's used.

Reply:
1000% this should not be here - just not 100% where to put it just yet

Reply:
I suppose it will be constant for most pages. Deal with it soon(_tm_).